### DISCUSSION TITLE
批量管理权限？

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#838] [Q&A] [answered]
最近QQ频繁被封，考虑关掉一部分不必要的功能，但似乎没有统一管理的方案？只能 XX权限 off 这样或者修改 authority.json，但修改数据库文件也无法方便的做到白名单或者全部禁用

**Comment by Arondight** [comment, marked as answer]
是不是需要一个默认行为列表？

**Reply by Xm798** [reply]
我觉得是的，然后  authority.json 覆盖默认权限

**Reply by Arondight** [reply]
> 我觉得是的，然后 authority.json 覆盖默认权限

加入有生之年计划（应该不会太久）
